bindings/python/setup.py.in

```diff
@@ -11,6 +11,6 @@ setup(
     license="GNU GPLv2+",
     author='Saggi Mizrahi',
     author_email='ficoos@gmail.com',
-    url='github.com/ficoos/ioprocess',
+    url='https://github.com/ficoos/ioprocess',
```

Comment:
Should we not point to this repository?

Reply:
I think so, but maybe worth to make it in a separate PR

Reply:
Worth making an additional commit for that? :)